.github/workflows/test.yml

```diff
@@ -27,7 +27,9 @@ jobs:
     uses: liquibase/build-logic/.github/workflows/os-extension-test.yml@main
     secrets: inherit
     with:
-      java: "[ 17, 21 ]"
+      # we support JDK 17, however opensearch-testcontainers used in the tests only supports JDK 21
+      javaBuildVersion: '17'
```

Comment:
@filipelautert: unlike what we discussed that doesn't work because it still also builds test sources and they don't compile with JDK17.

=> unless you see an alternative to just full-on drop JDK 17 support (or rewrite your test build logic) i'd suggest that we skip this upgrade until we're ready to drop JDK 17.

Reply:
I see, so I'm up to skipping it by now.